package.json

```diff
@@ -7,8 +7,10 @@
     "develop": "gulp develop",
     "build-dist-js": "gulp build-dist-js",
     "test": "gulp test-coverage",
-    "lint": "eslint src example --ext .jsx --ext .js"
-},
+    "lint": "eslint src example --ext .jsx --ext .js",
+    "prepublish": "gulp build-dist-scss"
+  },
+  "style": "./dist/css/react-calendar.css",
```

Comment:
Thanks for the clarification; this seems good to me. I'm happy to roll ahead with a minor release for this change.